contrib/automation_tests/orbit_bottom_up.py

```diff
@@ -33,7 +34,10 @@ def main(argv):
     test_cases = [
         ConnectToStadiaInstance(),
         FilterAndSelectFirstProcess(process_filter='hello_ggp'),
-        Capture(),
+        LoadSymbols(module_search_string="libc-2.24"),
```

Comment:
This looks a bit suspicious: below you also load the symbols for "hello_ggp". Is this intended?

Reply:
Yes, with frame pointers, we wont get any symbol information. The top-down/bottom-up view e2e tests validate the names of the root functions (clone, start, ioctl), and we need the names of those.